docs/guide/front-end-assets.md

```diff
@@ -68,18 +73,55 @@ The output will **sing** before it **dances**. If the modules are instantiated i
 
 the output will **dance** before it **sings**.
 
+**"Why do I have to export a function?"** A funny thing about JavaScript `import` statements: they don't guarantee any order in which the files are loaded. To fix that, Apostrophe requires you to export a function from each `ui/src/index.js` file so that it can call them in the order the modules are initialized.
+
+## Ordering the Sass output files
+
+The same rules apply to Sass files. Rules found in or imported by `ui/src/index.scss` files are compiled in the order the modules are activated in `app.js`. Of course, your Sass files should just contain CSS rules. Sass imports always execute in the order given, so you don't have to worry about exporting anything.
```

Comment:
No "Of courses." Let's chop that off. I'm confused about the exporting bit here. Is that because we're exporting in the JS files? I don't think export is a thing in Sass.

Reply:
It's not. I was trying to contrast it with the js, but it just confused things. I will shaddap about it.